App/App_macOS.swift

```diff
@@ -120,6 +120,10 @@ struct RootView: View {
                         if let windowNumber = window?.windowNumber {
                             browser.restoreByWindowNumber(windowNumber: windowNumber,
                                                           urlToTabIdConverter: navigation.tabIDFor(url:))
```

Comment:
Fallback, if nothing can be re-opened it should load the home page, instead of a blank page.